Limit translations to the text utilities (and lib/* and include/* ) only.
Set the message domain to text-utilities.

These changes will not be submitted upstream.

--- util-linux-2.33.2/po/POTFILES.in	2019-04-09 15:02:03.543017224 +0000
+++ util-linux-2.33.2/po/POTFILES.in.new	2019-05-16 08:40:25.696604939 +0000
@@ -1,30 +1,3 @@
-disk-utils/addpart.c
-disk-utils/blockdev.c
-disk-utils/cfdisk.c
-disk-utils/cramfs_common.c
-disk-utils/cramfs.h
-disk-utils/delpart.c
-disk-utils/fdformat.c
-disk-utils/fdisk.c
-disk-utils/fdisk.h
-disk-utils/fdisk-list.c
-disk-utils/fdisk-list.h
-disk-utils/fdisk-menu.c
-disk-utils/fsck.c
-disk-utils/fsck.cramfs.c
-disk-utils/fsck.minix.c
-disk-utils/isosize.c
-disk-utils/minix_programs.h
-disk-utils/mkfs.bfs.c
-disk-utils/mkfs.c
-disk-utils/mkfs.cramfs.c
-disk-utils/mkfs.minix.c
-disk-utils/mkswap.c
-disk-utils/partx.c
-disk-utils/raw.c
-disk-utils/resizepart.c
-disk-utils/sfdisk.c
-disk-utils/swaplabel.c
 include/all-io.h
 include/bitops.h
 include/blkdev.h
@@ -88,104 +61,6 @@ include/ttyutils.h
 include/widechar.h
 include/xalloc.h
 lib/blkdev.c
-libblkid/src/blkidP.h
-libblkid/src/cache.c
-libblkid/src/config.c
-libblkid/src/dev.c
-libblkid/src/devname.c
-libblkid/src/devno.c
-libblkid/src/encode.c
-libblkid/src/evaluate.c
-libblkid/src/getsize.c
-libblkid/src/init.c
-libblkid/src/llseek.c
-libblkid/src/partitions/aix.c
-libblkid/src/partitions/aix.h
-libblkid/src/partitions/atari.c
-libblkid/src/partitions/bsd.c
-libblkid/src/partitions/dos.c
-libblkid/src/partitions/gpt.c
-libblkid/src/partitions/mac.c
-libblkid/src/partitions/minix.c
-libblkid/src/partitions/partitions.c
-libblkid/src/partitions/partitions.h
-libblkid/src/partitions/sgi.c
-libblkid/src/partitions/solaris_x86.c
-libblkid/src/partitions/sun.c
-libblkid/src/partitions/ultrix.c
-libblkid/src/partitions/unixware.c
-libblkid/src/probe.c
-libblkid/src/read.c
-libblkid/src/resolve.c
-libblkid/src/save.c
-libblkid/src/superblocks/adaptec_raid.c
-libblkid/src/superblocks/apfs.c
-libblkid/src/superblocks/bcache.c
-libblkid/src/superblocks/befs.c
-libblkid/src/superblocks/bfs.c
-libblkid/src/superblocks/bitlocker.c
-libblkid/src/superblocks/btrfs.c
-libblkid/src/superblocks/cramfs.c
-libblkid/src/superblocks/ddf_raid.c
-libblkid/src/superblocks/drbd.c
-libblkid/src/superblocks/drbdmanage.c
-libblkid/src/superblocks/drbdproxy_datalog.c
-libblkid/src/superblocks/exfat.c
-libblkid/src/superblocks/exfs.c
-libblkid/src/superblocks/ext.c
-libblkid/src/superblocks/f2fs.c
-libblkid/src/superblocks/gfs.c
-libblkid/src/superblocks/hfs.c
-libblkid/src/superblocks/highpoint_raid.c
-libblkid/src/superblocks/hpfs.c
-libblkid/src/superblocks/iso9660.c
-libblkid/src/superblocks/isw_raid.c
-libblkid/src/superblocks/jfs.c
-libblkid/src/superblocks/jmicron_raid.c
-libblkid/src/superblocks/linux_raid.c
-libblkid/src/superblocks/lsi_raid.c
-libblkid/src/superblocks/luks.c
-libblkid/src/superblocks/lvm.c
-libblkid/src/superblocks/minix.c
-libblkid/src/superblocks/mpool.c
-libblkid/src/superblocks/netware.c
-libblkid/src/superblocks/nilfs.c
-libblkid/src/superblocks/ntfs.c
-libblkid/src/superblocks/nvidia_raid.c
-libblkid/src/superblocks/ocfs.c
-libblkid/src/superblocks/promise_raid.c
-libblkid/src/superblocks/refs.c
-libblkid/src/superblocks/reiserfs.c
-libblkid/src/superblocks/romfs.c
-libblkid/src/superblocks/silicon_raid.c
-libblkid/src/superblocks/squashfs.c
-libblkid/src/superblocks/stratis.c
-libblkid/src/superblocks/superblocks.c
-libblkid/src/superblocks/superblocks.h
-libblkid/src/superblocks/swap.c
-libblkid/src/superblocks/sysv.c
-libblkid/src/superblocks/ubi.c
-libblkid/src/superblocks/ubifs.c
-libblkid/src/superblocks/udf.c
-libblkid/src/superblocks/ufs.c
-libblkid/src/superblocks/vdo.c
-libblkid/src/superblocks/vfat.c
-libblkid/src/superblocks/via_raid.c
-libblkid/src/superblocks/vmfs.c
-libblkid/src/superblocks/vxfs.c
-libblkid/src/superblocks/xfs.c
-libblkid/src/superblocks/zfs.c
-libblkid/src/tag.c
-libblkid/src/topology/dm.c
-libblkid/src/topology/evms.c
-libblkid/src/topology/ioctl.c
-libblkid/src/topology/lvm.c
-libblkid/src/topology/md.c
-libblkid/src/topology/sysfs.c
-libblkid/src/topology/topology.c
-libblkid/src/topology/topology.h
-libblkid/src/verify.c
-libblkid/src/version.c
 lib/canonicalize.c
 lib/color-names.c
 lib/colors.c
@@ -194,28 +69,6 @@ lib/crc32.c
 lib/crc32c.c
 lib/env.c
 lib/exec_shell.c
-libfdisk/src/alignment.c
-libfdisk/src/ask.c
-libfdisk/src/bsd.c
-libfdisk/src/context.c
-libfdisk/src/dos.c
-libfdisk/src/fdiskP.h
-libfdisk/src/field.c
-libfdisk/src/gpt.c
-libfdisk/src/init.c
-libfdisk/src/item.c
-libfdisk/src/iter.c
-libfdisk/src/label.c
-libfdisk/src/partition.c
-libfdisk/src/parttype.c
-libfdisk/src/script.c
-libfdisk/src/sgi.c
-libfdisk/src/sun.c
-libfdisk/src/table.c
-libfdisk/src/test.c
-libfdisk/src/utils.c
-libfdisk/src/version.c
-libfdisk/src/wipe.c
 lib/fileutils.c
 lib/idcache.c
 lib/ismounted.c
@@ -228,32 +81,6 @@ lib/mbsalign.c
 lib/mbsedit.c
 lib/md5.c
 lib/monotonic.c
-libmount/python/context.c
-libmount/python/fs.c
-libmount/python/pylibmount.c
-libmount/python/pylibmount.h
-libmount/python/tab.c
-libmount/src/btrfs.c
-libmount/src/cache.c
-libmount/src/context.c
-libmount/src/context_loopdev.c
-libmount/src/context_mount.c
-libmount/src/context_umount.c
-libmount/src/fs.c
-libmount/src/init.c
-libmount/src/iter.c
-libmount/src/lock.c
-libmount/src/monitor.c
-libmount/src/mountP.h
-libmount/src/optmap.c
-libmount/src/optstr.c
-libmount/src/tab.c
-libmount/src/tab_diff.c
-libmount/src/tab_parse.c
-libmount/src/tab_update.c
-libmount/src/test.c
-libmount/src/utils.c
-libmount/src/version.c
 lib/pager.c
 lib/path.c
 lib/plymouth-ctrl.c
@@ -280,131 +107,7 @@ lib/sysfs.c
 lib/timer.c
 lib/timeutils.c
 lib/ttyutils.c
-libuuid/src/clear.c
-libuuid/src/compare.c
-libuuid/src/copy.c
-libuuid/src/gen_uuid.c
-libuuid/src/isnull.c
-libuuid/src/pack.c
-libuuid/src/parse.c
-libuuid/src/predefined.c
-libuuid/src/test_uuid.c
-libuuid/src/unpack.c
-libuuid/src/unparse.c
-libuuid/src/uuidd.h
-libuuid/src/uuid.h
-libuuid/src/uuidP.h
-libuuid/src/uuid_time.c
-login-utils/auth.c
-login-utils/auth.h
-login-utils/ch-common.c
-login-utils/ch-common.h
-login-utils/chfn.c
-login-utils/chsh.c
-login-utils/islocal.c
-login-utils/islocal.h
-login-utils/last.c
-login-utils/libuser.c
-login-utils/libuser.h
-login-utils/login.c
-login-utils/logindefs.c
-login-utils/logindefs.h
-login-utils/lslogins.c
-login-utils/newgrp.c
-login-utils/nologin.c
-login-utils/runuser.c
-login-utils/selinux_utils.c
-login-utils/selinux_utils.h
-login-utils/setpwnam.c
-login-utils/setpwnam.h
-login-utils/su.c
-login-utils/su-common.c
-login-utils/su-common.h
-login-utils/sulogin.c
-login-utils/sulogin-consoles.c
-login-utils/sulogin-consoles.h
-login-utils/utmpdump.c
-login-utils/vipw.c
-misc-utils/blkid.c
-misc-utils/cal.c
-misc-utils/fincore.c
-misc-utils/findfs.c
-misc-utils/findmnt.c
-misc-utils/findmnt.h
-misc-utils/findmnt-verify.c
-misc-utils/getopt.c
-misc-utils/kill.c
-misc-utils/logger.c
-misc-utils/look.c
-misc-utils/lsblk.c
-misc-utils/lsblk.h
-misc-utils/lsblk-mnt.c
-misc-utils/lsblk-properties.c
-misc-utils/lslocks.c
-misc-utils/mcookie.c
-misc-utils/namei.c
 misc-utils/rename.c
-misc-utils/test_uuidd.c
-misc-utils/uuidd.c
-misc-utils/uuidgen.c
-misc-utils/uuidparse.c
-misc-utils/whereis.c
-misc-utils/wipefs.c
-schedutils/chrt.c
-schedutils/ionice.c
-schedutils/taskset.c
-sys-utils/blkdiscard.c
-sys-utils/blkzone.c
-sys-utils/chcpu.c
-sys-utils/chmem.c
-sys-utils/choom.c
-sys-utils/ctrlaltdel.c
-sys-utils/dmesg.c
-sys-utils/eject.c
-sys-utils/fallocate.c
-sys-utils/flock.c
-sys-utils/fsfreeze.c
-sys-utils/fstrim.c
-sys-utils/hwclock.c
-sys-utils/hwclock-cmos.c
-sys-utils/hwclock.h
-sys-utils/hwclock-rtc.c
-sys-utils/ipcmk.c
-sys-utils/ipcrm.c
-sys-utils/ipcs.c
-sys-utils/ipcutils.c
-sys-utils/ipcutils.h
-sys-utils/ldattach.c
-sys-utils/losetup.c
-sys-utils/lscpu-arm.c
-sys-utils/lscpu.c
-sys-utils/lscpu-dmi.c
-sys-utils/lscpu.h
-sys-utils/lsipc.c
-sys-utils/lsmem.c
-sys-utils/lsns.c
-sys-utils/mount.c
-sys-utils/mountpoint.c
-sys-utils/nsenter.c
-sys-utils/pivot_root.c
-sys-utils/prlimit.c
-sys-utils/readprofile.c
-sys-utils/renice.c
-sys-utils/rfkill.c
-sys-utils/rtcwake.c
-sys-utils/setarch.c
-sys-utils/setpriv.c
-sys-utils/setsid.c
-sys-utils/swapoff.c
-sys-utils/swapon.c
-sys-utils/swapon-common.c
-sys-utils/swapon-common.h
-sys-utils/switch_root.c
-sys-utils/tunelp.c
-sys-utils/umount.c
-sys-utils/unshare.c
-sys-utils/wdctl.c
-sys-utils/zramctl.c
 term-utils/agetty.c
 term-utils/mesg.c
 term-utils/script.c
--- util-linux-2.33.2/configure	2019-04-09 15:02:09.715952416 +0000
+++ util-linux-2.33.2/configure.new	2019-05-16 08:42:10.403956353 +0000
@@ -5064,7 +5064,7 @@ fi
 
 
 # Define the identity of the package.
- PACKAGE='util-linux'
+ PACKAGE='text-utilities'
  VERSION='2.33.2'
 
 
